### PR TITLE
Explore: Implement feature toggle for logs sample

### PR DESCRIPTION
**Pull request by ivanahuckova**
This PR adds feature toggle for Logs sample. The feature toggle is enabled by default as the feature is stable, but we would like to give users a way how they can disable it, in case they need it.

### CODE REVIEW

**Comment by svennergr** [comment]
I am wondering, did you follow this:

```

// To change feature flags, edit:
//  pkg/services/featuremgmt/registry.go
// Then run tests in:
//  pkg/services/featuremgmt/toggles_gen_test.go
```

Because I would expect a change of `docs/sources/setup-grafana/configure-grafana/feature-toggles/index.md` as well, which is generated by running the test in `pkg/services/featuremgmt/toggles_gen_test.go`

**Comment by ivanahuckova** [comment]
@svennergr nice catch! Done.

**Review by svennergr — APPROVED** [review]

File: public/app/features/explore/Explore.tsx [hunk 1]
```diff
@@ -514,7 +514,9 @@ export class Explore extends React.PureComponent<Props, ExploreState> {
                             <ErrorBoundaryAlert>{this.renderFlameGraphPanel()}</ErrorBoundaryAlert>
                           )}
                           {showTrace && <ErrorBoundaryAlert>{this.renderTraceViewPanel()}</ErrorBoundaryAlert>}
-                          {showLogsSample && <ErrorBoundaryAlert>{this.renderLogsSamplePanel()}</ErrorBoundaryAlert>}
+                          {Boolean(config.featureToggles.logsSampleInExplore) && showLogsSample && (
```
Comment (svennergr) [inline public/app/features/explore/Explore.tsx:517]: One nit: `Boolean` shouldn't be needed.

Comment (ivanahuckova) [reply public/app/features/explore/Explore.tsx]: Fixed in https://github.com/grafana/grafana/pull/62291/commits/ce650bbd74ef5752b2a4869ed189e33f7949abba

File: docs/sources/setup-grafana/configure-grafana/feature-toggles/index.md [hunk 2]
```diff
@@ -30,6 +30,7 @@ Some stable features are enabled by default. You can disable a stable feature by
 | `internationalization`              | Enables internationalization                                                         | Yes                |
 | `accessTokenExpirationCheck`        | Enable OAuth access_token expiration check and token refresh using the refresh_token |                    |
 | `disablePrometheusExemplarSampling` | Disable Prometheus examplar sampling                                                 |                    |
+| `logsSampleInExplore`               | Access to logs sample feature in Explore                                             | Yes                |
```
Comment (chri2547) [inline docs/sources/setup-grafana/configure-grafana/feature-toggles/index.md:33]:
```suggestion
| `logsSampleInExplore`               | Enables access to the logs sample feature in Explore                                             | Yes                |
```

Comment (chri2547) [reply docs/sources/setup-grafana/configure-grafana/feature-toggles/index.md:33]: @ivanahuckova , does that ^^ wording work? Want to make sure I'm not changing the technical meaning. Thank you.

Comment (ivanahuckova) [reply docs/sources/setup-grafana/configure-grafana/feature-toggles/index.md:33]: Yes it does, but I had there `Enables logs sample feature in Explore` before and it got override by linting and changed it to `Access to logs sample feature in Explore`. Do you want me to try what you have suggested, to see if linter will be okay with it?

Comment (ivanahuckova) [reply docs/sources/setup-grafana/configure-grafana/feature-toggles/index.md]: Fixed in https://github.com/grafana/grafana/pull/62291/commits/1c0ddf27c786cc6f97dae2dd6bc7d4441de40b8a. Thank you so much for review and suggestion!

**Pull request merged**